### PR TITLE
Update "server" argument handling to also understand Victoria Metrica & Loki full HTTP API root when those are being hosted at non-default locations

### PR DESCRIPTION
**Pull request by jeanbaptistelab**

### CODE REVIEW

**Comment by troelsjessen** [comment]
As discussed verbally, we can change the interface so that the user still give the URL as <address> argument, and the slash command will check if the address starts with http[s]:// and handle accordingly.

**Comment by jeanbaptistelab** [comment]
> As discussed verbally, we can change the interface so that the user still give the URL as
> argument, and the slash command will check if the address starts with http[s]:// and handle accordingly.

This is now implemented with the last commits

File: src/victoria_metrics.c [hunk 1]
```diff
@@ -127,12 +150,12 @@ void * vm_push(void * arg) {
 
         curl_easy_setopt(curl, CURLOPT_POSTFIELDSIZE, buffer_size);
         curl_easy_setopt(curl, CURLOPT_POSTFIELDS, buffer);
+        curl_easy_setopt(curl, CURLOPT_FAILONERROR, 1);
         res = curl_easy_perform(curl);
         if (res != CURLE_OK) {
             printf("Failed push: %s\n", curl_easy_strerror(res));
-        } else {
-            buffer_size = 0;
```
Comment (edvardxyz) [inline src/victoria_metrics.c:134]: Why this change?
Before I had this so that a pc with connection drop could use up the buffer until connection is back

Comment (jeanbaptistelab) [reply src/victoria_metrics.c:134]: I think I did that so that debug prints wouldn't flood the output while I was testing the Docker version of Victoria Metrics. But now that this is in place, there's no need to actually change the current behaviour, I'll put it back in as it was.

**Review by edvardxyz — APPROVED** [review]
